Assignment 3-JS/script/script.js

```diff
@@ -0,0 +1,312 @@
+      and the pen is in your hands. It is your chance 
+      to write a beautiful story for yourself. Happy New Year.`
+    },
+    "january-26": {
+      title: " Republic day",
+      date: " 26 January ",
+      discription: `Our nation is the greatest country in 
+      the world, but let that never stop us from striving to 
+      make it even better. I wish you a very Happy Republic Day!`
+    },
+    "february-14": {
+      title: " Baby's Birthday",
+      date: " 14 Fabruary ",
+      discription: `I feel so lucky to have you as my friend.
+      Hope your birthday is as special as you are.May all of 
+      your dreams come true. Thanks for being such a great 
+      friend. Happy birthday!`
+    },
+    "october-2": {
+      title: " Gandhi Jayanti ",
+      date: " 2 October ",
+      discription: `A man is the sum of his actions,
+      of what he has done, of what he can do, nothing 
+      else. Happy Gandhi Jayanti`
+    },
+    "august-15": {
+      title: " Indipendence Day",
+      date: " 15 August ",
+      discription: `Freedom is the way God intended us; it 
+      is something we are born with. Something that no one can
+      take away from you. Let’s celebrate Freedom! Happy Independence Day.`
+  
+    },
+    "june-5": {
+      title:" World's Environment Day",
+      date: " 5 June ",
+      discription:`For most of history, man has had to fight 
+      nature to survive; in this century he is beginning to 
+      realise that, in order to survive, he must protect it.`
+    }
+  }
+   
+  const renderCalendar = (month,year) => {
+   
+    document.getElementsByClassName('month-name')[0].innerHTML = monthArray[month];
+    document.getElementsByClassName('current-year-name')[0].innerHTML = year;
+    
+  
+    const firstDay = new Date(year, month,1).getDay();//to get the first day of current month
+    const lastDay = new Date(year,month + 1, 0).getDate();//to get the last day of current month
+  
+    const lastDaysIndex = new Date(year,month+1,0).getDay();//to get the  starting days of next month
+    const nextDays = 7 - lastDaysIndex - 1;//calculate the starting days of next month
+    
+    const prevLastDays = new Date(year,month,0).getDate();//get the last days of previous month
+    
+    //get the week number at first date of a month
+    var dateinput1 = new Date(year,month,1);
+    var weekOfStartDate = new Date(dateinput1.getFullYear(),dateinput1.getMonth(),dateinput1.getDate());
+    var startDate = weekOfStartDate.getWeek();
+  
+    //get the week number at last date of a month
+    var dateinput2 = new Date(year,month + 1, 0)
+    var weekOfEndDate = new Date(dateinput2.getFullYear(),dateinput2.getMonth(),dateinput2.getDate());
+    var endDate = weekOfEndDate.getWeek();
+  
+    let week = "";
+    for(let j = startDate; j <= endDate; j++) { 
+      week += `<div class="week">${j}</div>`;
+    }
+    weekDays.innerHTML = week;
+  
+    month_days.innerHTML = "";

```

Comment:
Use variable name in camel-case format.

Reply:
Changed it to monthDays and also changed other variable names.